lib/modules/manager/asdf/extract.spec.ts

```diff
@@ -303,6 +304,13 @@ dummy 1.2.3
             packageName: 'java-jdk',
             depName: 'java',
           },
+          {
+            currentValue: '1.7',
```

Comment:
this probably needs a different versioning. did you tested it on a real repo?

Reply:
Ah. No, not yet (as per the checkboxes in the PR description).

Anything in particular you foresee?

Whenever I find the time I'll learn how to try it on a local repo, and verify.

Reply:
The repo includes tags like `jq-1.7rc2`. `semver` won't work for short versions. `semver-coerced` might work, but I don't know if it recognizes those rc ones properly or if it coerces them into a stable version (which would be a problem). `loose` wouldn't recognize or sort properly

Reply:
How does one (unit) test this with Renovate, that it'll discover versions correctly? Any ideas on how to adjust this PR to solve this?

Reply:
run on a test repo

Reply:
Each of Renovate's versioning schemes includes its own unit tests, so you need to pick one which suits the data/versions that you'll be receiving

Reply:
loose versioning might work

Reply:
@MPV do you have time to test this?